devel/cutter-r2/files/Cutter

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+EXECDIR=$(dirname "$0")
+if [ "$@" == *"--pythonhome"*]; then
```

Comment:
There needs to be a space before the closing `]`; otherwise:
```
line 3: [: missing `]'
```

Also, `"$@"` expands to each argument quoted. An `if` statement expects exactly one argument. So this will fail if you specify no arguments to the script:

```
line 3: [: ==: unary operator expected
```

And if you specify more than one argument:

```
line 3: [: too many arguments
```

You want `"$*"` here to always give you a single argument containing the expansion of all arguments.

`==` isn't POSIX sh compatible. You want `=` instead.

Finally, I don't think you can check for a glob match using the `[` command; I think you can only check for string equality. But you can do a glob match using a case statement. So I suggest this replacement script:

```
#!/bin/sh

EXECDIR=$(dirname "$0")

case "$*" in
    *--pythonhome*)
        "$EXECDIR/Cutter.bin" "$@"
        ;;
    *)
        "$EXECDIR/Cutter.bin" --pythonhome "@PYFRAMEWORK@/Versions/@PYVER@/" "$@"
        ;;
esac
```